nautobot_circuit_maintenance/templates/nautobot_circuit_maintenance/rawnotification.html

```diff
@@ -59,6 +59,10 @@ <h1>{{ object.subject }}</h1>
             {% else %} {{ object.parsed }} {% endif %}
           </td>
         </tr>
+        <tr>
+          <td>Stamp</td>
+          <td>{{ object.stamp }}</td>
+        </tr>
         <tr>
           <td>Date</td>
```

Comment:
Can we make these more descriptive to the user? Perhaps "Notification Timestamp" and "Processing Date" or something like that would be accurate and more descriptive?